### DISCUSSION TITLE
Tekla Structures connection

### DISCUSSION THREAD

**Discussion by MatthiasRimm** [BHoM/BHoM#1432] [General]
Hello!
Are there any plans for having a connection to Tekla Structures in the future? At the country I'm working in, most structural engineering firms are or already have moved over to it from Revit.

**Comment by FraserGreenroyd** [comment]
Hi @MatthiasRimm I have just found and made public a version of [TeklaStructures_Toolkit](https://github.com/BHoM/TeklaStructures_Toolkit) which we started a few years ago but haven't been able to take further due to resource constraints for other parts of the BHoM projects. I went through it last night and updated it a little bit to make it compile against the latest BHoM/BHoM_Engine, but there is still an issue around an outdated `AdapterIdName` property, which @alelom has raised an issue for [here](https://github.com/BHoM/TeklaStructures_Toolkit/issues/62) and hopes to resolve shortly. We don't currently have any plans in our roadmap to develop this toolkit up further, but are very happy to support others in the community if they wish to pick it up 😄 It would be great if you would like to get involved and help develop this further with the interoperability between the BHoM and TeklaStructures!

**Reply by MatthiasRimm** [reply]
Thank you for making it available and updating it. Sadly I only have basic knowledge in Python so I am not up to the task of developing it further. Hopefully in the future!
